en_US/faq/use-guide.md

```diff
@@ -417,3 +417,141 @@ Client disconnect link error code list:
+
+## Why are there client IDs like "CENSYS" or other unfamiliar clients?
+
+**Tags:** [*Security*](tags.md#security)
+
+CENSYS is an internet scanning and reconnaissance tool that periodically scans the IPv4 address space to discover default ports for protocols such as HTTP, SSH, MQTT, etc. So if you find MQTT clients with a client ID of "CENSYS" or other unknown clients accessing your MQTT broker, it indicates that you are currently at a relatively lower level of security protection. The following measures can effectively help you avoid this issue:
+
+1. Avoid using default configurations, such as the AppID and AppSecret used for verifying HTTP API access permissions in EMQX.
+2. Enable authentication, which can be password-based authentication or JWT authentication, to prevent situations where only knowledge of an IP address is required for login.
+3. Enable TLS mutual authentication, allowing only clients with valid certificates to access the system.
+4. Enable authorization to prevent unauthorized devices from accessing sensitive data after logging in.
+5. Configure your firewall to close unnecessary ports as much as possible.
+
+## Can I shared subscribe to the system topic?
+
+**Tags:** [*Shared Subscription*](tags.md#shared-subscription)
+
+Yes, some system messages, such as client online/offline events, may be published frequently, so using a shared subscription is essential for clients. Example subscription: `$share/group1/$SYS/brokers/+/clients/+/connected`.
+
+## Why can't I receive retained messages when using shared subscriptions?
+
+**Tags:** [*Shared Subscription*](tags.md#shared-subscription)
+
+The MQTT protocol stipulates that when the client uses a shared subscription, the server cannot send retained messages to it.
+
+## Why are messages occasionally lost when using shared subscriptions?
+
+**Tags:** [*Shared Subscription*](tags.md#shared-subscription)
+
+When the connection of the shared subscriber is disconnected but the session still exists, the server will continue delivering the messages to the shared subscriber, and these messages will be temporarily stored in the corresponding session. This will cause other online shared subscribers to appear to have not consumed all the messages. In addition, if the shared subscriber chooses to create a new session when reconnecting, the messages cached in the old session will be permanently lost.
+
+If we confirm that the above situation does not exist, but the problem of message loss still exists, then we can use the client tracking function of EMQX for further investigation.
+
+## What should I do if EMQX prompts that the port is occupied (eaddrinuse) when starting?
```

Comment:
```suggestion
## What should I do if EMQX reports a "port is occupied (eaddrinuse)" error during startup?
```

Reply:
“port is occupied” is not an actual log.